components/Features/FeatureForm.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styles from './FeatureForm.module.scss';
+
+export default function FeatureForm() {
+  return (
+    <div className={styles['form-layout']}>
+      <div className={styles['form-block']}>
+        <h2 className={styles['form-header']}>
+          Increase your
+          <br />
+          charity donations
+        </h2>
+        <div className={styles['form-text']}>
+          Leave your contact information below, and we&apos;ll be in touch soon
+          to help you get started
+        </div>
+        <form
+          id="contact-form"
+          name="email-form"
+          className={styles.form}
+          method="post"
+        >
+          <label htmlFor="name-2" className={styles['field-label']}>
+            Name
+            <input
+              type="text"
+              className={[styles['name-text-field'], styles['w-input']].join(
+                ' ',
+              )}
+              maxLength="256"
+              name="name"
+              id="name-2"
+              placeholder="Name"
+            />
+          </label>
+
+          <label htmlFor="email-2" className={styles['field-label']}>
+            Email Address
+            <input
+              type="email"
+              className={[styles['name-text-field'], styles['w-input']].join(
+                ' ',
+              )}
+              maxLength="256"
+              name="email"
+              id="email-2"
+              placeholder="Email Address"
+            />
+          </label>
+          <input
+            id="contact-form-btnSubmit"
```

Comment:
I don't think we need `id`'s in this entire application, since we will use on click callbacks directly in each component, let me know if we do need this though.

Reply:
sounds good I was following the mock so I didn't know if it was necessary or not